[2024-10-23T00:31:28.919+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-23T00:31:28.963+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aula_4_1_get_stocks_dag.MSFT scheduled__2022-01-27T00:00:00+00:00 [queued]>
[2024-10-23T00:31:28.986+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aula_4_1_get_stocks_dag.MSFT scheduled__2022-01-27T00:00:00+00:00 [queued]>
[2024-10-23T00:31:28.987+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-23T00:31:29.027+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): MSFT> on 2022-01-27 00:00:00+00:00
[2024-10-23T00:31:29.056+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=311) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-23T00:31:29.062+0000] {standard_task_runner.py:72} INFO - Started process 368 to run task
[2024-10-23T00:31:29.077+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'aula_4_1_get_stocks_dag', 'MSFT', 'scheduled__2022-01-27T00:00:00+00:00', '--job-id', '337', '--raw', '--subdir', 'DAGS_FOLDER/aula_4_1_get_stocks.py', '--cfg-path', '/tmp/tmpl7a9vhuy']
[2024-10-23T00:31:29.092+0000] {standard_task_runner.py:105} INFO - Job 337: Subtask MSFT
[2024-10-23T00:31:30.099+0000] {task_command.py:467} INFO - Running <TaskInstance: aula_4_1_get_stocks_dag.MSFT scheduled__2022-01-27T00:00:00+00:00 [running]> on host 375c06cd6615
[2024-10-23T00:31:31.929+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='aula_4_1_get_stocks_dag' AIRFLOW_CTX_TASK_ID='MSFT' AIRFLOW_CTX_EXECUTION_DATE='2022-01-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-27T00:00:00+00:00'
[2024-10-23T00:31:31.947+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-23T00:31:31.999+0000] {cache.py:130} INFO - Failed to create TzCache, reason: Error creating TzCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. TzCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-23T00:31:32.000+0000] {cache.py:336} INFO - Failed to create CookieCache, reason: Error creating CookieCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 13] Permission denied: '/home/***/.cache/py-yfinance'. CookieCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2024-10-23T00:32:19.950+0000] {base.py:128} ERROR - Failed to get ticker 'MSFT' reason: HTTPSConnectionPool(host='query1.finance.yahoo.com', port=443): Max retries exceeded with url: /v1/test/getcrumb (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fdcecf03020>: Failed to resolve 'query1.finance.yahoo.com' ([Errno -3] Temporary failure in name resolution)"))
[2024-10-23T00:32:40.962+0000] {history.py:123} ERROR - $MSFT: possibly delisted; no timezone found
[2024-10-23T00:32:40.984+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-23T00:32:43.756+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-23T00:32:43.757+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=aula_4_1_get_stocks_dag, task_id=MSFT, run_id=scheduled__2022-01-27T00:00:00+00:00, execution_date=20220127T000000, start_date=20241023T003128, end_date=20241023T003243
[2024-10-23T00:32:43.773+0000] {job.py:229} INFO - Heartbeat recovered after 75.10 seconds
[2024-10-23T00:32:44.040+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-23T00:32:44.060+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
